Core/Utilities/Graphs/GraphData.py

```diff
@@ -401,7 +401,7 @@ def initialize( self ):
       self.keys = self.sortKeys()
 
     self.values = [ self.parsed_data[k] for k in self.keys ]
-    self.errors = [ self.parsed_errors[k] for k in self.keys ]
+    self.errors = [ self.parsed_errors[k] if k in self.parsed_errors else 0.0 for k in self.keys ]
     values_to_sum = [ self.parsed_data[k] for k in self.keys if k != '' ]
```

Comment:
and also to this line:
values_to_sum = [ self.parsed_data[k] if k in self.parsed_data else 0.0 for k in self.keys if k != '' ]